src/pybind/mgr/dashboard/controllers/crush_rule.py

```diff
@@ -4,14 +4,28 @@
 from cherrypy import NotFound
 
 from . import ApiController, ControllerDoc, RESTController, Endpoint, ReadPermission, \
-    UiApiController
+    UiApiController, EndpointDoc
 from ..security import Scope
 from ..services.ceph_service import CephService
 from .. import mgr
 
 
+LIST_SCHEMA = {
+    "rule_id": (int, 'Rule ID'),
```

Comment:
Why not trying [something more structured](https://www.attrs.org/en/stable/why.html#readability) (as [`dataclasses`](https://docs.python.org/3/library/dataclasses.html) are not available in 3.6)?